pd/src/main.rs

```diff
@@ -48,9 +50,31 @@ enum Command {
         metrics_port: u16,
     },
 
-    /// Prints a sample `app_data` JSON object that can act as a template for
-    /// editing genesis configuration.
-    CreateGenesisTemplate,
+    /// Generates a directory structure containing necessary files to run a
+    /// testnet based on input configuration.
+    GenerateTestnet {
+        /// How many validator nodes to create configuration for.
+        #[structopt(short, long)]
+        num_validator_nodes: usize,
+        /// Number of blocks per epoch.
+        #[structopt(short, long)]
+        epoch_duration: u64,
+        /// Path to CSV file containing initial allocations.
+        #[structopt(short, long, parse(from_os_str))]
+        allocations_input_file: PathBuf,
+        /// Path to JSON file containing initial validator configs.
+        #[structopt(short, long, parse(from_os_str))]
+        validators_input_file: PathBuf,
+        /// Path to directory to store output in. Must not exist.
+        #[structopt(short, long, parse(from_os_str))]
+        output_dir: PathBuf,
+        /// Testnet name, e.g. `penumbra-euporie`
+        #[structopt(short, long)]
+        chain_id: String,
+        /// IP Address to start `tendermint` nodes on. Increments by three to make room for `pd` and `postgres` per node.
+        #[structopt(short, long, default_value = "192.167.10.2")]
+        starting_ip: Ipv4Addr,
+    },
```

Comment:
Should we specify default values for these fields?  

My sense is that: (a) we'll probably have more parameters in the future, and people might not want to specify all of them, and that (b) if we do have default values, it's better for them to be in the Rust source than in a wrapper script.

For instance, down below in the script, it plugs in 60 as a default value, but if we keep the defaults here, they'll all be in one place and we just have one thing to edit, and the default is provided to all of the users of the program, not just users of the script.

Reply:
I'll set the defaults to be the latest testnet configurations.